Open Judge System/OJS.Services/Business/Participants/ParticipantsBusinessService.cs

```diff
@@ -66,6 +66,28 @@ public bool CanCompeteByContestAndUserId(Contest contest, string userId)
             return participant;
         }
 
+        public void ChangeTimeForActiveInOnlineContestByContestIdAndMinutes(
```

Comment:
Please, try finding a better name. I have been on the meetings where we decided what the functionality will be and yet I do not understand what the method does by its name...

Reply:
`ExtendContestEndTimeForAllActiveParticipantsByContestByParticipantContestStartTimeRangeAndTimeIntervalInMinutes`